src/app/components/calendar/calendarForm.tsx

```diff
@@ -25,34 +26,27 @@ const CalendarForm = () => {
     color: "blackAndWhite",
   };
 
-  const [calendarSetup, setCalendar] = useState<CalendarValuesProps | null>(null);
-
-  const [calendarDates, setCalendarDates] = useState({
+  const calendarDatesDefault = {
     weekNumber: null,
     weekDayNumbers: [],
     month: new Date().getMonth(),
     year: new Date().getFullYear(),
-  });
+  }
+
+  const [calendarSetup, setCalendar] = useLocalStorage('calendarData', calendarDefaultValues)
+  const [calendarDates, setCalendarDates] = useLocalStorage('calendarDatesData', calendarDatesDefault)
+
+  const [mount, setMount]= useState(false)
```

Comment:
What is the purpose of the `mount` variable`?

Reply:
I've tried to resolve this issue, however, I still got a loader component and also warning message in the console. According to this article, I think there is no better option than to set something like "mounting" useState. 🤔 

https://dev.to/gvegacl/nextjs-server-prop-did-not-match-client-prop-and-conditional-styleclass-rendering-2i4c